sphinx/source/docs/user_guide/geo.rst

```diff
@@ -11,8 +11,9 @@ Tile provider maps
 ------------------
 
 Bokeh is compatible with several XYZ tile services that use the Web Mercator projection.
-The module :ref:`bokeh.tile_providers` contains several pre-configured tile sources with appropriate attribution.
-To add these to a plot, use the method :func:`~bokeh.models.plots.Plot.add_tile`.
+Bokeh uses `xyzservices`_ library to take care of the tile sources and their attributions.
+To add these to a plot, use the method :func:`~bokeh.models.plots.Plot.add_tile`. You can pass
+any name xyzservices may recognize. ``retina`` keyword can control the resolution of tiles.
```

Comment:
Is it possible you are missing a "The" in your last sentence? Also, it would be great to be more precise about how/where to include ``retina``. Do I understand correctly that I can include ``retina`` in the string, i.e. ``add_tile("CartoDB Positron retina")``? Or should I rather use the optional Boolean ``retina`` argument? Since retina is not used in ``geo_tile_source.py``, I'd also recommend putting this additional information about retina below the ``.. bokeh-plot`` element.

Reply:
Well, yes you can include `retina` in the string to use retina tiles (if available) but I am not sure how much should that be a documented part of the API. I have included it for the backwards compatibility with your existing `Vendors` that make a distinction between normal and retina tiles. So I would say, use `retina` argument if possible but if you pass it in string, we do that one step for you.

Reply:
I have added a bit on the `retina` keyword and included it in the code snippet. Check if it is the approach you like.